### DISCUSSION TITLE
게임내의 유저데이터 분석및 보상이가능할까요?

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#555] [5. Q&A]
현재 서비스 중인게임에서, 유저가 IWP 결제를 했는데, 보상이 지급되지 않았다는 제보를 받았습니다.

아마 유저의 착각일 가능성이 크긴하지만, 차후에라도, 문제가 있을수 있어서..

1. 실제 특정유저가 어떤 아이템을 결제했는지 확인할수 있는 방법이 있나요?
2. 현재 데이터를 서버에 저장하고있는데, 저장내역(시간대별 로그)을 확인할수 있는방법이 있나요?
3. 특정유저의 데이터에 접근해 데이터를 임의로 변경(보상)할수 있는 방법이 있을까요?
4. 더불어 게임내의 유저의 달성내역(몇스테이지까지 갔고, 어디쯤에서 유저이탈이 일어나는지 알고 싶습니다.)을 위한 애널리틱스 같은기능이 있을까요?(아니면 업데이트 예정이라도..)

이상입니다.
4가지 문의에대한 답변 부탁드려요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 
서버 로그는 확인하는 기능을 개발 중입니다. 업데이트 시기는 미정인 점 양해 부탁 드립니다.
클라 로그는 기존 앱 로그 확인하시는 것과 동일한 방법으로 로그 확인이 가능합니다. (안드로이드는 logcat을 이용하는 등)
감사합니다.
